core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -95,6 +101,16 @@ private[spark] abstract class WebUI(
     serverInfo.foreach(_.addHandler(handler, securityManager))
   }
 
+  /** Attaches a handler to this UI. */
+  def attachHandler(contextPath: String,
```

Comment:
This is not the right style. See the class's constructor for an example.

Reply:
@vanzin I have run scalafmt on modified class and updated the PR. Is it ok now.?